tethysext/atcore/models/file_database/meta_mixin.py

```diff
@@ -0,0 +1,45 @@
+"""
+********************************************************************************
+* Name: file_database.py
+* Author: glarsen
+* Created On: November 2, 2020
+* Copyright: (c) Aquaveo 2020
+********************************************************************************
+"""
+import json
+import logging
+import os
+
+from sqlalchemy import Column
+from sqlalchemy.dialects.postgresql import JSON
+
+log = logging.getLogger('tethys.' + __name__)
+
+
+class MetaMixin(object):
+    meta = Column('metadata', JSON)
```

Comment:
Move the Column definition of meta back to the classes where they were defined before (FileCollection and FileDatabase) and define the meta property on the MetaMixin with a simple default (e.g. meta = dict()). Then the MetaMixin could be used on non-model classes.